go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/megaease/easegress/v2
 
-go 1.23.0
+go 1.24
```

Comment:
Don't upgrade golang version in a huge feature. It should be a dedicated one.

Reply:
Done